app/src/main/java/de/danoeh/antennapod/preferences/PreferenceUpgrader.java

```diff
@@ -117,5 +120,15 @@ private static void upgrade(int oldVersion, Context context) {
                 prefs.edit().putString(UserPreferences.PREF_DRAWER_FEED_COUNTER, "2").apply();
             }
         }
+        if (oldVersion < 2070095) {
```

Comment:
Please add this to the `if` statement above - it should be migrated when switching to the next version, not when switching to the current version.